pkgs/tools/misc/mongodb-tools/goconvey.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, goPackages, fetchFromGitHub, callPackage }:
```

Comment:
Any particular reason you're making your own `goconvey` as opposed to using (or updating) the one in `go-packages.nix`?